app/Notifications/RequestVoucherForApprovalNotification.php

```
@@ -3,18 +3,17 @@
-use App\Models\CashRequest;
+use App\Models\Voucher;
 use Illuminate\Bus\Queueable;
 use Notification;
 
-class RequestCashVoucherForApprovedNotification extends Notification
+class RequestVoucherForApprovalNotification extends Notification
 {
     use Queueable;
     private $token;
     private $model;
     public $id;
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)